pull_into_place/commands/09_compare_best_designs.py

```diff
@@ -504,6 +504,43 @@ def score_value(self, design):
 
 
 
+class ExtraFilterHandler (Metric):
+    """
+    Class that handles filters input by the user. The user should put a "+" or "-"
+    at the beginning of each filter name to define whether larger numbers (+) or
+    smaller numbers (-) are better. Used to make an excel column for
+    extra filters.
+    """
+
+    def __init__(self):
+        align = 'center'
+        num_format = '0.000'
+
+    def get_name(self,filtername):
+
+        if filtername[0] == '+' or filtername[0] == '-':
+            self.color = True
+        else:
+            self.color = False
+
+        self.title = filtername
+        return self.title,self.color
```

Comment:
I really like the idea of encoding how the filter should be interpreted in its name, but I think it might be worth having a slightly more general syntax than just a +/- prefix.  For example, maybe for some filters you just want to be close to a specific value, rather than positive or negative.  Or maybe at some point we'll want to figure out things like `num_format` or `width` from the filter name.

The syntax I have in mind is `Arbitrary Filter Name [+]`.  The part between the square brackets is what we'll interpret.  For now we can just look for `+` or `-`, but in the future it wouldn't be hard to cram more stuff in there.

We should also add a function to parse these names, e.g. `parse_filter_name(name)`.  It could go in `structures.py`.  It would take a filter name like `Arbitrary Filter Name [+]`, use regular expressions to extract various fields from it, then return a dictionary like:

    {'name': 'Arbitrary Filter Name', 'direction': 1}

This function would avoid some code duplication between here and `plot_funnels`.